### PR TITLE
Issue and PR templates

### PR DESCRIPTION
**Pull request by danez**
This is the starting point for Issue and pr templates.

Maybe also a table in the PR template for the env information?

### CODE REVIEW

**Comment by hzoo** [comment]
We should make it correspond to the labels (which we can also change https://github.com/babel/babel/blob/master/lerna.json#L6-L14) https://github.com/babel/babel/labels

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!---
+BEFORE YOU SUBMIT please read the following:
+
+If you have a support request or question please submit them to on of this resources:
+
+* Discussion Forum: https://discuss.babeljs.io/
+* StackOverflow: http://stackoverflow.com/questions/tagged/babeljsusing the tag `[babeljs]`
```
Comment (hzoo) [inline .github/ISSUE_TEMPLATE.md]: needs a space here right?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Slack community: https://slack.babeljs.io/
+
+Issues on github are only issues related to development of babelitself.
```
Comment (hzoo) [inline .github/ISSUE_TEMPLATE.md]: here too

**Comment by hzoo** [comment]
Is it worth mentioning contributing or linking the readme?

**Comment by hzoo** [comment]
You can just make it commit if you want, and use `[skip ci]` if you want too

**Comment by danez** [comment]
I changed the PR template to link to the Contributing.md and the issue template to link to the readme.md. 
Fixed all the typos, spaces, etc. and made the version info a table.

**Comment by tarlepp** [comment]
+1 for this.

**Comment by hzoo** [comment]
Aside: just 👍 with reactions please.

How would we go about asking for the versions of the packages? Since `Babel` version is really the babel-core version.

It might be a lot of info but I think we also care about the versions of any relevant babel-* pkgs

**Comment by hzoo** [comment]
Oh another suggestion: instead of just input/output/config - even making a reproducible repo or the simplest is to just use the REPL and post a link to it. When we are able to import any plugin into the repl that would be even better (https://github.com/babel/babel.github.io/issues/858)

**Comment by danez** [comment]
About the babel version: is there a command with npm maybe to ask for all versions of all installed babel-packages? like `npm ls babel*` that would be nice.

**Pull request merged**